_posts/2018-08-13-this-week-in-rust-wasm-006.md

```diff
@@ -0,0 +1,53 @@
+---
+title:  "This Week in Rust and WebAssembly 006"
+---
+
+Hello and welcome to another issue of *This Week in Rust and WebAssembly*!
+
+[Rust](https://rust-lang.org) is a systems language pursuing the trifecta: safety, concurrency, and speed.
+
+[WebAssembly](http://webassembly.org) is a stack-based virtual machine and instruction set. It is fast, safe, portable, and part of the open Web platform. By compiling to WebAssembly, we can run Rust code on the Web!
+
+This is a weekly summary of Rust and WebAssembly's progress and community.
+
+Did we miss something? Tweet to us at [@rustwasm](https://twitter.com/rustwasm) or [send us a pull request](https://github.com/rustwasm/rustwasm.github.io).
+
+**Want to get involved in Rust and WebAssembly? [Join the Rust and WebAssembly working group!][get-involved]**
+
+## New Tools, Crates, and Projects
+
+* [Railway diagram generator for Rust macro_rules! macros with a wasm demo](https://www.reddit.com/r/rust/comments/96q6jb/show_reddit_a_syntaxdiagram_generator_for_macro/)
+* [A work-in-progress pull request to expose Wasm SIMD intrinsics to stdsimd](https://github.com/rust-lang-nursery/stdsimd/pull/549)
+* WIP prototype of using DWARF debug info in Firefox's debugger.html to debug inlined stack frames in Rust+Wasm programs
+  * [gif](https://drive.google.com/file/d/1Yf1gXBSWNdSzXomy8GKUOiPuZVFAASfw/view)
+  * WIP spec work: [here](https://yurydelendik.github.io/webassembly-dwarf/) and [here](https://gist.github.com/yurydelendik/802f36983d50cedb05f984d784dc5159)
+  * WIP branches: [here](https://github.com/yurydelendik/debugger.html/tree/x-scopes) and [here](https://github.com/yurydelendik/emscripten/tree/x-scopes)
+
+## News and Releases
+
+* [An official subcharter of the W3C wasm WG for debugging has been created](https://github.com/WebAssembly/debugging)
+
+## Articles, Blog Posts, and Talks
+
+* Cleaned up, curated and opinionated "Tools" section into "Tools You Should Know" [section](https://rustwasm.github.io/book/tools.html) to the book
```

Comment:
s/and opinionated/and added an opinionated
s/into/into the
s/to the book/as part of the book